pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
It sure is a 100% improvement to the non-working code right now :)
But really fixing this might be harder than expected...